tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml

```diff
@@ -14,7 +14,7 @@ data:
   {{ $key }}: {{ $value }}
   {{- end }}
   MARKER_CMD: {{ .Values.testData.markerCmd}}
-  TEST_MARKER: {{ .Values.testData.marker }}
+  TEST_MARKER: "{{ .Values.testData.marker }}"
```

Comment:
any reason to surround only the test marker by quotes? is it to support the `and` keywords?

Reply:
Indeed, the reason for surrounding the test marker with quotes is to support keyword expressions like aws and not aws_s3. By enclosing the expression in quotes, it ensures that the entire expression is treated as a single argument. Without the quotes, the command could potentially interpret the expression as multiple arguments, leading to unexpected behavior.